src/support/alloy_rlp.rs

```diff
@@ -77,6 +77,13 @@ impl<const BITS: usize, const LIMBS: usize> Decodable for Uint<BITS, LIMBS> {
     #[inline]
     fn decode(buf: &mut &[u8]) -> Result<Self, Error> {
         let bytes = Header::decode_bytes(buf, false)?;
+
+        // We only need to check if the first byte is zero to make sure there are no
+        // leading zeros
```

Comment:
Comment should mention that this is according to the RLP spec.

Reply:
quoted directly